Makefile

```diff
@@ -99,11 +99,34 @@ doc-open:
 size:
 	find . -name "esp-build.map" -exec ${IDF_PATH}/tools/idf_size.py {} \;
 
-build-esp32-bin:
-	cargo +esp espflash save-image --package micro-rdk-server --merge --chip esp32 target/xtensa-esp32-espidf/micro-rdk-server-esp32.bin -T micro-rdk-server/esp32/partitions.csv -s 4mb  --bin micro-rdk-server-esp32 --target=xtensa-esp32-espidf  -Zbuild-std=std,panic_abort --release
+build-esp32-bin: build-esp32-ota
+	cargo +esp espflash save-image \
```

Comment:
These are looking much better. A few small things:

- Order of `--bin` and `--partition-table` args is not consistent between the two lists.
- Let's move any "only in one list" flags/args to the end of each. So for `build-esp32-bin` you already have `--merge` there, but `--flash-size` is in the middle.
- Let's make the actual target file always the last argument.

So the order of the 'sections' is:
- identical
- common flag but different args (e.g. partition table)
- unique (e.g. `--merge` and `--flash-size`)
- file target

Reply:
removed `--flash-size` as it doesn't appear to be necessary, both `make build-*` still work.

Reply:
put it back, turns out it did influence the final build size for some reason. It appears like it may be adding padding with `--flash-size` for the merged binary to be the same size as the flash target. building without is only slightly larger than the ota app image by itself.